core/src/components/button/test/spinner/button.e2e.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ios', 'md', 'ionic-md', 'ionic-ios'] }).forEach(({ config, screenshot, title }) => {
```

Comment:
Since the functionality doesn't change, we don't need to test for all the modes. Make sure to remove the snapshots for `md` and `ionic-ios`.

```suggestion
configs({ modes: ['ios', 'ionic-md'] }).forEach(({ config, screenshot, title }) => {
```